### PR TITLE
idr0038 DOI and example image

### PR DESCRIPTION
**Pull request by eleanorwilliams**
Added the links to the DOIs for the study and the three experiments.  Added a link to a good example image from experiment A.  

No testing needed as only the study file is changed, just checking that the changes look ok.  The DOIs have been added manually to prod44.

### CODE REVIEW

**Comment by jburel** [comment]
Does it mean that the tag prep during the release process is invalidated?
I have only pushed them to my branch, just trying to clarify the workflow

cc @joshmoore @sbesson

**Comment by sbesson** [comment]
In the past, my understanding is that we tagged what was deployed and the DOI additions were effectively part of the next tag. As the public tags have not been pushed yet and this captures the state of current idr.openmicroscopy.org, I would be fine with creating a new tag including this.

File: idr0038-held-kidneylightsheet/idr0038-study.txt [hunk 1]
```diff
@@ -46,9 +46,11 @@ Term Source URI	http://purl.obolibrary.org/obo/	http://www.ebi.ac.uk/efo/	http:/
 				
 Experiment Number	1			
 Comment[IDR Experiment Name]	idr0038-held-kidneylightsheet/experimentA			
+Experiment Data Publisher	University of Dundee			
+Experiment Data DOI	https://dx.doi.org/10.17867/10000110a			
 Experiment Description	"We have adapted the mouse kidney rudiment assay to generate renal organoids. To demonstrate that the organoid culture method of re-aggregated kidney rudiments in PDMS discs resulted in 3D kidney organoids with organotypic structures, we stained fixed six day old organoids for various developmental markers and imaged them with a light sheet fluorescent microscope. We have stained for the following markers: Megalin, Laminin, Cytokeratin, Pax2, Six2, Wt1, Synaptopodin and Nephrin and have also used the lectin stain PNA (peanut agglutinin). The spheroids were usually imaged as z stacks from five angles."			
 Experiment Size	5D Images: 	Average Image Dimension (XYZCT): X: 1920 Y: 1902 Z: 570 C: 4  V: 5 T: 1	Total Tb: 1	# fill in any values you know
-Experiment Example Images				
+Experiment Example Images	https://idr.openmicroscopy.org/webclient/?show=image-3414064	endpoint imaging Six2/100-U-13-Six2-PNA-Synpo-MV-with-beads.czi [0]
```
Comment (sbesson) [inline idr0038-held-kidneylightsheet/idr0038-study.txt]: Is that captured anywhere in the UI?

Comment (eleanorwilliams) [reply idr0038-held-kidneylightsheet/idr0038-study.txt]: No, not currently.

**Pull request merged**